src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionEditArtwork.jest.tsx

```diff
@@ -556,4 +502,4 @@ const mockArtwork = {
   title: "Untitled",
   width: "11",
   " $fragmentType": "MyCollectionArtworkForm_artwork",
-} as MyCollectionArtworkForm_artwork$data
+} as CleanRelayFragment<MyCollectionEditArtwork_artwork$data>
```

Comment:
@olerichter00 Could you, please, explain this change?

Reply:
With this helper, it's possible to type the mock data without having to specify the following fields: `"$refType" | " $fragmentRefs" | " $fragmentSpreads" | " $fragmentType"`.

Reply:
Got it, thanks